kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3Storage.java

```diff
@@ -75,25 +74,31 @@ public S3Storage(S3SinkConnectorConfig conf, String url) {
    */
   public AmazonS3 newS3Client(S3SinkConnectorConfig config) {
     ClientConfiguration clientConfiguration = newClientConfiguration(config);
+
     AmazonS3ClientBuilder builder = AmazonS3ClientBuilder.standard()
-                                        .withAccelerateModeEnabled(
-                                            config.getBoolean(WAN_MODE_CONFIG)
-                                        )
-                                        .withPathStyleAccessEnabled(true)
-                                        .withCredentials(config.getCredentialsProvider())
```

Comment:
I think I'd prefer to add a credentials recommender that is `Configurable` and accepts the key pair, than define these two config properties at the top level. Choosing to use this recommender should make this choice more explicit and will be consistent with how credentials are setup with this connector. Otherwise, we add an additional if/else branch in the credentials configuration which might not be as simple to explain. 

See https://github.com/confluentinc/kafka-connect-storage-cloud/blob/master/kafka-connect-s3/src/test/java/io/confluent/connect/s3/DummyAssertiveCredentialsProvider.java for a simple example of what you'd like to introduce here and https://github.com/confluentinc/kafka-connect-storage-cloud/pull/230 for a PR that adds another credentials recommender.

Reply:
Never mind. 
Let's keep the separate top-level configs to conform with what other aws connectors define now.